src/main/java/com/twingly/search/domain/Post.java

```diff
@@ -300,4 +397,221 @@ public void setBlogRank(int blogRank) {
     public void setTags(List<String> tags) {
         this.tags = tags;
     }
+
+    /**
+     * Gets id
+     *
+     * @return the id
+     */
+    public String getId() {
+        return id;
+    }
+
+    /**
+     * Sets id.
+     *
+     * @param id the id
+     */
+    public void setId(String id) {
+        this.id = id;
+    }
+
+    /**
+     * Gets author
+     *
+     * @return the author
+     */
+    public String getAuthor() {
+        return author;
+    }
+
+    /**
+     * Sets author
+     *
+     * @param author the author
+     */
+    public void setAuthor(String author) {
+        this.author = author;
+    }
+
+    /**
+     * Gets location code
+     *
+     * @return the location code
+     */
+    public String getLocationCode() {
+        return locationCode;
+    }
+
+    /**
+     * Sets location code
+     *
+     * @param locationCode the location code
+     */
+    public void setLocationCode(String locationCode) {
+        this.locationCode = locationCode;
+    }
+
+    /**
+     * Gets inlinks count
+     *
+     * @return the inlinks count
+     */
+    public int getInlinksCount() {
+        return inlinksCount;
+    }
+
+    /**
+     * Sets inlinks count
+     *
+     * @param inlinksCount the inlinks count
+     */
+    public void setInlinksCount(int inlinksCount) {
+        this.inlinksCount = inlinksCount;
+    }
+
+    /**
+     * Gets re-indexed at date
+     *
+     * @return the re-indexed at date
+     */
+    public Date getReindexedAt() {
+        return reindexedAt;
+    }
+
+    /**
+     * Sets re-indexed at date
+     *
+     * @param reindexedAt the re-indexed at date
+     */
+    public void setReindexedAt(Date reindexedAt) {
+        this.reindexedAt = reindexedAt;
+    }
+
+    /**
+     * Gets links
+     *
+     * @return the links
+     */
+    public List<String> getLinks() {
+        if (links == null) {
+            links = new ArrayList<>();
+        }
+        return links;
+    }
+
+    /**
+     * Sets links
+     *
+     * @param links the links
+     */
+    public void setLinks(List<String> links) {
+        this.links = links;
+    }
+
+    /**
+     * Gets images
+     *
+     * @return the images
+     */
+    public List<String> getImages() {
+        if (images == null) {
+            images = new ArrayList<>();
+        }
+        return images;
+    }
+
+    /**
+     * Sets images
+     *
+     * @param images the images
+     */
+    public void setImages(List<String> images) {
+        this.images = images;
+    }
+
+    /**
+     * Gets coordinates
+     *
+     * @return the coordinates
+     */
+    public String getCoordinates() {
```

Comment:
@roback There was just no info about how coordinates are represented.
I'll add mapping for coordinate a bit later.